react/src/components/base/Typography/Typography.stories.mdx

```diff
@@ -23,97 +22,32 @@ const slugifyStyle = (style) => {
-
-### Usage
-
-Texta font has the following font weight options:
-
-| Font weight | Specs |
-| :--- | :--- |
-| Light | 300 |
-| Medium | 500 |
-| Bold | 700 |
-| Black | 800 |
-
-All font files can be downloaded either from the [**Mayflower Github repo**](https://github.com/massgov/mayflower/tree/develop/assets/fonts) or from the [**Mayflower NPM package**](https://unpkg.com/@massds/mayflower@6.0.0/fonts/). Please read the web font [**web font license**](https://www.fontspring.com/lic/htswufoczd) **\*\*carefully to proceed**.\*\*
-
-### License
-
-Fontspring grants Licensee a perpetual, worldwide, non-exclusive and non-transferrable license to install the Texta as webfont **on Websites that are owned by the Commonwealth of Massachusetts** using the @font-face selector in CSS files. For other usage, you may have to acquire an additional license through Fontsprint.
-
-For more details about Texta's terms of use, please refer to our [**web font license**](https://www.fontspring.com/lic/htswufoczd).
-
-
-
-<div className="page-content">
-  <Heading text="Source Code Pro" level={1} />
-  <SidebarHeading title="Characters" level={2} />
-  <div className="sb-block sb-block--mono">
-    <div className="sb-text">{letters.toUpperCase()}</div>
-    <div className="sb-text">{letters}</div>
-    <hr />
-    <div className="sb-text">{numbers}</div>
-    <hr />
-    <div className="sb-text">{characters}</div>
-    <hr />
-  </div>
-  <div className="sb-block">
-    Source Code Pro is an open source font available on Google Fonts.
-    <DecorativeLink
-      text="Download the fonts or see more information on Google Fonts"
-      href="https://fonts.google.com/specimen/Source+Code+Pro"
-    />
-  </div>
-
-  <Heading text="Texta" level={1} />
-  <SidebarHeading title="Characters" level={2} />
-  <div className="sb-block">
-    <div className="sb-text">{letters.toUpperCase()}</div>
-    <div className="sb-text">{letters}</div>
-    <hr />
-    <div className="sb-text">{numbers}</div>
-    <hr />
-    <div className="sb-text">{characters}</div>
-    <hr />
-  </div>
-  <SidebarHeading title="Styles" level={2} />
-  <div className="sb-block sb-grid">
-    {
-      styles.map((style) => {
-        const className = `ma-texta-${slugifyStyle(style)}`;
-        return(
-          <React.Fragment>
-            <span className={className}>{style}</span>
-            <span className="markdown-body">
-              <code>{`.${className}`}</code>
-            </span>
-          </React.Fragment>
-        );
-      })
-    }
-  </div>
-  <SidebarHeading title="Sizes" level={2} />
-  <div className="sb-block">
-    { Object.keys(typographyOptions.size).map((key) => {
-        const Element = typographyOptions.size[key];
-        return(
-          <Element>{key}</Element>
-        );
-      })
-    }
-  </div>
-</div>
+<Canvas>
```

Comment:
Can you remove the code block and make this story look at the one for Texta?
![Screen Shot 2020-09-25 at 1 06 18 PM](https://user-images.githubusercontent.com/5789411/94296295-9dd95300-ff30-11ea-824d-e9c4949eee28.png)

And also order this story after Texta
![Screen Shot 2020-09-25 at 1 04 42 PM](https://user-images.githubusercontent.com/5789411/94296332-ad589c00-ff30-11ea-9d8f-0b1bd59eab74.png)

Reply:
@smurrayatwork ^

Reply:
I will fix this in DP-20050-font-switch